src/client/sandbox/event/focus-blur.js

```diff
@@ -172,7 +203,25 @@ export default class FocusBlurSandbox extends SandboxBase {
     }
 
     static getInternalEventFlag (type) {
-        return type === 'focus' ? INTERNAL_FOCUS_FLAG : INTERNAL_BLUR_FLAG;
+        return INTERNAL_FOCUS_BLUR_FLAG_PREFIX + type;
+    }
+
+    static getFocusBlurEventType (focusInOutEventType) {
+        if (focusInOutEventType === 'focusin')
+            return 'focus';
+        else if (focusInOutEventType === 'focusout')
+            return 'blur';
+
+        return null;
+    }
+
+    static getFocusInOutEventType (focusBlurEventType) {
+        if (focusBlurEventType === 'focus')
+            return 'focusin';
+        else if (focusBlurEventType === 'blur')
```

Comment:
>  getFocusBlurEventType
>  getFocusInOutEventType
>  These two methods look very similar and confusing

I need to perform two way convesion:
`blur <-> focusout`
`focus <-> focusin`

Propose a better way by your opinion